RoboJackets/urc-software
# [sim] add terrain, rehaul drivebase model

# Description
This PR does the following:
- Fixes the dynamics of the rocker-bogie joints on the rover. They are still a little unstable, but much more realistic than before.
- Imports the ERC Mars terrain into our Gazebo simulation.
- Changes how we launch things. There's now two launch scripts. `bringup.launch.py` will launch all nodes for the physical robot while `bringup_simulation.launch.py` will launch nodes for simulation.

# Testing Steps (if relevant)

1. `colcon build`
2. `. install/setup.bash`
3. `ros2 launch urc_bringup bringup_simulation.launch.py`

Expectation: Should launch the Gazebo simulation with the rover model and new terrain. Publishing to `rover_drivetrain_controller/cmd_vel` will send velocity commands to the rover.

# Self Checklist
